jina/hubble/helper.py

```diff
@@ -250,3 +251,40 @@ def upload_file(
     response = getattr(requests, method)(url, data=data, headers=headers, stream=stream)
 
     return response
+
+
+def disk_cache(
```

Comment:
Let's do this one more explicit and do not let add `Exceptions` as one wants.

Handling `exceptions` as normal behavior is a little dangerous and while it makes sense for this case, I would prefer to discourage it and specialize better this decorator

Reply:
So this means hardcoding the exception right ?
Should I rename the decorator ?

Reply:
done